src/Benchmarks/Configuration/Scenarios.cs

```diff
@@ -155,14 +155,14 @@ public Scenarios(IScenariosConfiguration scenariosConfiguration)
 
         public int Enable(string partialName)
         {
-            if(string.Equals(partialName, "[default]", StringComparison.OrdinalIgnoreCase))
+            if (string.Equals(partialName, "[default]", StringComparison.OrdinalIgnoreCase))
             {
                 EnableDefault();
                 return 2;
             }
 
             var props = typeof(Scenarios).GetTypeInfo().DeclaredProperties
-                .Where(p => string.Equals(partialName, "[all]", StringComparison.OrdinalIgnoreCase) || p.Name.StartsWith(partialName, StringComparison.OrdinalIgnoreCase))
+                .Where(p => string.Equals(partialName, "[all]", StringComparison.OrdinalIgnoreCase) || p.Name.IndexOf(partialName, StringComparison.OrdinalIgnoreCase) >= 0)
```

Comment:
Shoot that's going to break many of the TechEmpower scripts but I understand the "enable the minimum" motivation. ☹️ but 🆗